easybuild/easyconfigs/n/ncurses/ncurses-5.9-CrayIntel-2015.11-XC.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncurses'
+version = '5.9'
+
+homepage = 'http://www.gnu.org/software/ncurses/'
+description = """The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,
+ and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
+ function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
+
+toolchain = {'name': 'CrayIntel', 'version': '2015.11-XC'}
+toolchainopts = {'optarch': True, 'shared': True, 'pic': True}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+patches = ['ncurses-%(version)s_configure_darwin.patch']
```

Comment:
a bit weird to have this here (this patch is specific to OS X)
 Add a line note